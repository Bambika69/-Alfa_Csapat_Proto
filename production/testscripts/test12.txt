createfield -t safehouse
createequipment axe -s SH1
createvirologist SH1
startturn V1
search
pickupequipment A1