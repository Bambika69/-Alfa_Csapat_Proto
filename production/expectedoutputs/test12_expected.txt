Field created:
ID: SA1
Type: Safehouse
Neighbours: null

Equipment created:
ID: E1 
Type: Axe
Destination: SA1
Result: Successful


Virologist created:
ID: V1
Field: SA1
Backpack: B1

V1â€™s turn started.

  Virologist searched:
Field: SA1
Type: Safehouse
Neighbours: null
Virologists: V1
GeneticCodes: null
Equipments: E1
Elements: null

Equipment added to inventory:
Virologist: V1
Equipment: E1
Field: SA1
Result: Successful
